models/migrations/v186.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"xorm.io/xorm"
+)
+
+func addOrgIDHookTaskColumn(x *xorm.Engine) error {
```

Comment:
You don't need that, since webhook table has already the column.

Reply:
@lunny  But then how to find undelivered orgs task?
see ``services/webhook/deliver.go``